src/container/rpc.h

```diff
@@ -358,6 +361,15 @@ CRT_RPC_DECLARE(cont_tgt_epoch_aggregate, DAOS_ISEQ_CONT_TGT_EPOCH_AGGREGATE,
 CRT_RPC_DECLARE(cont_tgt_snapshot_notify, DAOS_ISEQ_CONT_TGT_SNAPSHOT_NOTIFY,
 		DAOS_OSEQ_CONT_TGT_SNAPSHOT_NOTIFY)
 
+#define DAOS_ISEQ_CONT_PROP_SET	/* input fields */		 \
```

Comment:
(style)  Macros with complex values should be enclosed in parentheses

Reply:
False alarm